### PR TITLE
GH action to merge the stable to the master branch automatically

### LINKED ISSUE #6316: Merge stable to master branch automatically
## Provide a description of the task

Often times we forget to merge the `stable` branch to the `master` after pushing some changes to the former.

It's not rare that it results with unexpected and time inefficient conflicts.

If it's automated then this risk is greatly minimized, and we're sure that the `master` branch contains what it should contain.

## 📃 Other details

*   Browser: Any

### PR DESCRIPTION
**Pull request by mlewand**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Internal (ckeditor5): GH action to merge the stable to the master branch automatically. Closes #6316.

---

### Additional information

Notes can be found in this comment: https://github.com/ckeditor/ckeditor5/issues/6316#issuecomment-630112493

### CODE REVIEW

File: .github/workflows/merge-stable.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        env:
+          GITHUB_TOKEN: ${{ secrets.STABLE_MERGE_GITHUB_TOKEN }}
+      # Report errors if any
+      - uses: rtCamp/action-slack-notify@v2.0.2
+        id: error_message_slack
+        name: Slack notification
+        if: (steps.merge_action.outputs.status != 201) && (steps.merge_action.outputs.status != 204)
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_CHANNEL: "cke5-code"
```
Comment (Reinmar) [inline .github/workflows/merge-stable.yml:28]: Could we get this to cke5-ci? I think that cke5-code is mostly dead.

**Comment by Reinmar** [comment]
Just to be sure – if the merge fails (conflicts), then nothing happens and we need to resolve this manually, right?

File: .github/workflows/merge-stable.yml (hunk 1 above)
Comment (mlewand) [reply .github/workflows/merge-stable.yml:28]: Wanted to bring it for a discussion.

I made it `cke5-code` temporarily. For me `cke5-ci` sounds best, good visibility and not creating a dedicated channel for something that should happen _extremely rarely_.

Comment (mlewand) [reply .github/workflows/merge-stable.yml:28]: Wanted to bring it for a discussion.

I made it `cke5-code` temporarily. For me `cke5-ci` sounds best, good visibility and not creating a dedicated channel for something that should happen _extremely rarely_.

**Comment by mlewand** [comment]
> Just to be sure – if the merge fails (conflicts), then nothing happens and we need to resolve this manually, right?

That's right.

**Pull request merged**